### PR TITLE
feat: General pytorch function

### PR DESCRIPTION
**Pull request by 578223592**
https://github.com/lf-edge/ekuiper/issues/2810

### CODE REVIEW

File: extensions/functions/mnist/mnist.go [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+	// Create the output tensor
+	output, e := ort.NewEmptyTensor[float32](f.outputShape)
+	if e != nil {
+		return fmt.Errorf("error creating output tensor: %w", e), false
+	}
+	defer output.Destroy()
+
+	// The input and output names are required by this network; they can be
+	// found on the MNIST ONNX models page linked in the README.
+	session, e := ort.NewAdvancedSession(f.modelPath,
```
Comment (ngjaying) [inline extensions/functions/mnist/mnist.go:106]: The names are hard coded now. Will it work for any model?

**Comment by 578223592** [comment]
@ngjaying a small question about gci:
i have already install gci and run `gci write --skip-generated -s standard -s default -s "prefix(github.com/lf-edge/ekuiper)" .` like:
image
but the check still show:
e34db773165f3f63521dc22a23cb105

File: extensions/functions/onnx/onnx.conf [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# include onnx c api
+/usr/local/onnx/lib
```
Comment (ngjaying) [inline extensions/functions/onnx/onnx.conf:2]: Users need to install lib here?

File: extensions/functions/onnx/onnx.go [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+			value, err := cast.ToFloat64Slice(arg, cast.CONVERT_SAMEKIND, cast.IGNORE_NIL)
+			if err != nil {
+				return fmt.Errorf("invalid %d parameter, expect float64 but got %[2]T(%[2]v) with err %v", i, args[i], err), false
+			}
+			input, err := ort.NewTensor(inputInfo.Dimensions, value)
+			if err != nil {
+				return fmt.Errorf("convert to onnx tensor failed with err %v", err), false
+			}
+			inputTensors = append(inputTensors, input)
+		case ort.TensorElementDataTypeFloat: // onnx的float转换到go为float32
```
Comment (ngjaying) [inline extensions/functions/onnx/onnx.go:88]: Translate Chinese comment

File: extensions/functions/onnx/onnx.go [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+			valueI64, err := cast.ToInt64Slice(arg, cast.CONVERT_SAMEKIND)
+			if err != nil {
+				return fmt.Errorf("invalid %d parameter, expect int64 but got %[2]T(%[2]v) with err %v", i, args[i], err), false
+			}
+			input, err := ort.NewTensor(inputInfo.Dimensions, valueI64)
+			if err != nil {
+				return fmt.Errorf("convert to onnx tensor failed with err %v", err), false
+			}
+			inputTensors = append(inputTensors, input)
+		case ort.TensorElementDataTypeUint64: // todo:这些tflite的类型待测试
```
Comment (ngjaying) [inline extensions/functions/onnx/onnx.go:134]: Another chn comment

File: extensions/functions/onnx/onnx.json [hunk 5]
```diff
@@ -0,0 +1,77 @@
+{
+  "about": {
+    "trial": false,
+    "author": {
```
Comment (ngjaying) [inline extensions/functions/onnx/onnx.json:4]: Can add your name here.

File: extensions/functions/onnx/onnx.go [hunk 6]
```diff
@@ -0,0 +1,323 @@
+// Copyright 2021-2024 EMQ Technologies Co., Ltd.
```
Comment (ngjaying) [inline extensions/functions/onnx/onnx.go:1]: Copyright can be your email. This is not a property of EMQ

**Review by ngjaying — CHANGES_REQUESTED** [review]

File: extensions/functions/onnx/onnx.conf (hunk 2 above)
Comment (578223592) [reply extensions/functions/onnx/onnx.conf:2]: This is not required, but it can also be installed in this directory. Perhaps I should remove it? @ngjaying

**Comment by 578223592** [comment]
@ngjaying the problem of gci still bothers me.
the gci show：
```
  Running [/home/runner/golangci-lint-1.61.0-linux-amd64/golangci-lint run  --out-${NO_FUTURE}format line-number] in [/home/runner/work/ekuiper/ekuiper] ...
  internal/binder/mock/mock_factory.go:20: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/lf-edge/ekuiper/v2) (gci)
  	"github.com/lf-edge/ekuiper/contract/v2/api"
  extensions/functions/accumulateWordCount/accumulateWordCount.go:21: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/lf-edge/ekuiper/v2) (gci)
  	"github.com/lf-edge/ekuiper/contract/v2/api"
  pkg/message/mock.go:18: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/lf-edge/ekuiper/v2) (gci)
  	"github.com/lf-edge/ekuiper/contract/v2/api"
  Error: issues found
  Ran golangci-lint in 89659ms
```
i found ` extensions/functions/accumulateWordCount/accumulateWordCount.go:21: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/lf-edge/ekuiper/v2) (gci)`
but when i run gci ,nothing changed , like:
```
➜  ekuiper git:(torch_dev_swx) gci write --skip-generated -s standard -s default -s "prefix(github.com/lf-edge/ekuiper)"  extensions/functions/accumulateWordCount/accumulateWordCount.go
➜  ekuiper git:(torch_dev_swx) git status
位于分支 torch_dev_swx
您的分支与上游分支 'origin/torch_dev_swx' 一致。

无文件要提交，干净的工作区
➜  ekuiper git:(torch_dev_swx) 
```

**Review by ngjaying — APPROVED** [review]

**Pull request merged**